README.md

```diff
@@ -20,4 +20,302 @@ composer require bedita/php-sdk
+        'filter' => ['parent' => 'some-parent-id'],
+    ];
+    $response = (array)$client->getObjects('documents', $query); // arguments passed are string $type and array $query
+
+    // get documents using a filter and more query options
+    // include: including related data by relations "has_media" and "part_of"
+    // lang: the desired lang (if translation is available)
+    $query = [
+        'filter' => ['parent' => 'some-parent-id'],
+        'include' => 'has_media,part_of',
+        'lang' => 'en_EN',
+    ];
+    $response = (array)$client->getObjects('documents', $query); // arguments passed are string $type and array $query
+```
+
+```php
+    // get document 17823
+    $response = (array)$client->getObject(17823, 'documents'); // arguments passed are int|string $id, string $type
+
+    // get document 17823, english version, including related data by relations "has_media" and "part_of"
+    $query = [
+        'lang' => 'en_EN',
+        'include' => 'has_media,part_of',
+    ];
+    $response = (array)$client->getObject(17823, 'documents', ['lang' => 'en_EN']); // arguments passed are int|string $id, string $type and array $query
+
+    // get folder website-footer
+    $response = (array)$client->getObject('website-footer', 'folders'); // arguments passed are int|string $id, string $type
+```
+
+You can also use `get(string $path, ?array $query = null, ?array $headers = null)` to retrieve a list of resources or objects or a single resource or object
+
+```php
+    // get api status
+    $response = (array)$client->get('/status'); // argument passed is string $path
+
+    // get api home info
+    $response = (array)$client->get('/home'); // argument passed is string $path
+
+    // get a list of documents
+    $query = [
+        'filter' => ['parent' => 'some-parent-id'],
+    ];
+    $response = (array)$client->get('/documents', $query); // arguments passed are string $path and array $query
+
+    // get a stream by id
+    $response = (array)$client->get('/streams/the-stream-id'); // argument passed is string $path
+```
+
+When you need to get relationships related data, you can you `getRelated(int|string $id, string $type, string $relation, ?array $query = null, ?array $headers = null)`.
+
+```php
+    // get subfolders for a folder with ID 888
+    $query = [
+        'filter' => ['type' => 'folders'],
+    ];
+    $subfolders = $client->getRelated(888, 'folders', 'children', $query); // arguments passed are int|string $id, string $type, string $relation, array $query
+
+    // get children (all types) inside a folder with unique name 'my-folder-uname'
+    $children = $client->getRelated('my-folder-uname', 'folders', 'children'); // arguments passed are int|string $id, string $type, string $relation
+```
+
+### Save data
+
+You can use `save(string $type, array $data, ?array $headers = null)` or `saveObject(string $type, array $data, ?array $headers = null)` (deprecated, use `save` instead) to save data.
```

Comment:
don't mention `saveObject` since its depreceted.
I would add:

This is the standard method to use to create or update objects or resources in BEdita 4.
Please note that the payload is made of object `attributes`, without the need to specify a `data.attribute` section that you must use when dealing with direct POST o PATCH API calls.